modules/apps/portal/portal-servlet-test/src/testIntegration/java/com/liferay/portal/servlet/filters/i18n/test/I18nFilterTest.java

```diff
@@ -206,6 +250,16 @@ private String _getPrependI18nLanguageId(
 				_mockHttpServletResponse.getCookies());
 		}
 
+		PortalInstances.getCompanyId(_mockHttpServletRequest);
+
```

Comment:
@ChrisKian, please remove `PortalInstances.getCompanyId(_mockHttpServletRequest);`.  Since VirtualHost related tests's cookieLocale is not null and _language.updateCookie() will invoke `PortalInstances.getCompanyId(_mockHttpServletRequest)`